.travis/test.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/sh -ex
 
 rm -rf ovs
-git clone --depth 1 -b branch-2.12 https://github.com/openvswitch/ovs.git
+rm -rf ovn
+
+git clone --depth 1 -b master https://github.com/openvswitch/ovs.git
+git clone --depth 1 -b  branch-20.06 https://github.com/ovn-org/ovn.git
```

Comment:
Thanks for taking this up. Can we use master for ovn too because pinning to 20.06 will again result in new feature miss? Also, you can refer to sample build script to configure ovs/ovn seperately as source referring to https://github.com/ovn-org/ovn/blob/master/.travis/linux-build.sh .

Reply:
I updated to pull ovn from master.

I am not sure why splitting ovn and ovs setup would be beneficial... There is no real setup for ovn while setting up ovs the database schema is taken from the ovn repo to apply to the ovs dbs.

Reply:
Thanks. Yeah that is also valid point as for sandbox we don't really need all of it.